pkg/alertmanager/multitenant.go

```diff
@@ -479,10 +479,46 @@ func (am *MultitenantAlertmanager) ServeHTTP(w http.ResponseWriter, req *http.Re
 	userAM, ok := am.alertmanagers[userID]
 	am.alertmanagersMtx.Unlock()
 
-	if !ok || !userAM.IsActive() {
+	if !ok && am.fallbackConfig != "" {
```

Comment:
I would move this logic into a dedicate function, used to create the fallback config.